pkgs/top-level/all-packages.nix

```diff
@@ -17916,7 +17916,8 @@ in
   inherit (callPackages ../servers/mpd { })
     mpd mpd-small mpdWithFeatures;
 
-  mpd_clientlib = callPackage ../servers/mpd/clientlib.nix { };
+  mpd_clientlib = libmpdclient;
```

Comment:
Maybe we want to move this to alias.nix.

Reply:
I will redo all these references in a future "treewide".